src/main/util.ts

```diff
@@ -37,5 +39,23 @@ export const mkdir = (dirPath: string) => {
   }
 };
 
+export const handleOSQuery: () => OperatingSystems | null = () => {
+  const isDarwin = os.platform() === OperatingSystems.MACOS;
+  const isWindows = os.platform() === OperatingSystems.WINDOWS;
+  const isLinux = os.platform() === OperatingSystems.LINUX;
+
+  if (isDarwin) {
+    return OperatingSystems.MACOS;
+  }
+  if (isWindows) {
+    return OperatingSystems.WINDOWS;
+  }
+  if (isLinux) {
```

Comment:
This isn't quite what I meant, I was more referring to using a map/object so you don't need three if statements, something like this:

```
const platformToOperatingSystem = {
    'darwin': OperatingSystems.MACOS,
    'windows': OperatingSystems.WINDOWS,
    'linux': OperatingSystems.LINUX
};

return platformToOperatingSystem[os.platform()] || null;
```

But - don't worry about it, this is approach is fine. There's a fine line between giving a review and saying "do it how I'd do it". There is nothing wrong with the approach you've taken given the number of options, just consider the object approach if you encounter this problem again with say, 5-10 options as it will cut down on if statements :) no need to implement this